lua/entities/gmod_wire_expression2/core/interpolations.lua

```diff
@@ -0,0 +1,14 @@
+__e2setcost(2)
+
+--This seemed faster then manually typing out all 30 e2functions
+local from_easings = {"OutElastic","OutCirc","InOutQuint","InCubic","InOutCubic","InOutBounce","InOutSine","OutQuad","InOutCirc","InElastic","OutBack","InQuint","InSine","InBounce","InQuart","OutSine","OutExpo","InOutExpo","InQuad","InOutElastic","InOutQuart","InExpo","OutCubic","OutQuint","OutBounce","InCirc","InBack","InOutQuad","OutQuart","InOutBack"}
+
+for k, v in pairs(from_easings) do
```

Comment:
if youre gonna add all of them anyway just replace this with `pairs(math.ease)` and you're done
only drawback would be if a new function was added and it had different parameters I guess